testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/xxe/SecureProcessing2Test.java

```diff
@@ -230,6 +231,7 @@ public void after() throws Exception {
      *                "resteasy.document.expand.entity.references" is set to default value
      * @tpSince RESTEasy 3.0.16
      */
+    //@Ignore //rls
```

Comment:
Please remove this commented line, here and in all similar cases below in the same class